dts/bindings/memory-controllers/atmel,sam-smc.yaml

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2022, Basalte bv
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Atmel Static Memory Controller (SMC).
+
+  The SMC allows to interface with static-memory mapped external devices such as
+  SRAM, NOR Flash, NAND Flash, SDRAM...
```

Comment:
Use the full list: SRAM, PSRAM, PROM, EPROM, EEPROM, LCD Module, NOR Flash and NAND Flash.

Drop SDRAM because there is a dedicated controller to handle that.